src/pkg/caendr/caendr/services/cloud/cloudrun.py

```diff
@@ -100,7 +100,7 @@ def get_job_execution_status(name):
 
   # TODO: Make sure this is adequate
   for condition in response['conditions']:
-    if condition['type'] == 'Completed':
+    if condition['type'] == 'Completed' and condition['state'] == 'CONDITION_SUCCEEDED':
```

Comment:
whenever possible, prefer class ENUMS instead of literal strings for logic control. Eg: `Status.COMPLETED` instead of `'Completed'`. (applies to `'CONDITION_SUCCEEDED'` as well.